project/CopyrightHeader.scala

```diff
@@ -133,5 +133,3 @@ trait CopyrightHeader extends AutoPlugin {
 }
 
 object CopyrightHeader extends CopyrightHeader
-
-object CopyrightHeaderInPr extends CopyrightHeader
```

Comment:
This should have been deleted along with the change in https://github.com/apache/incubator-pekko/commit/bcec7bae0768cf9b3c9715fce3f908437752524b#diff-068e87d37c866101b33e15bb754968e8e77fd43f90f2d3ad50344742b9604113L110-L111 , its now no longer doing anything